Solutions/Trend Micro Apex One/Data Connectors/TrendMicro_ApexOne.json

```diff
@@ -0,0 +1,126 @@
+{
+    "id": "TrendMicroApexOne",
+    "title": "Trend Micro Apex One",
+    "publisher": "Trend Micro",
+    "descriptionMarkdown": "The [Trend Micro Apex One](https://www.trendmicro.com/en_us/business/products/user-protection/sps/endpoint.html) data connector provides the capability to ingest [Trend Micro Apex One events](https://docs.trendmicro.com/en-us/enterprise/trend-micro-apex-central-2019-online-help/appendices/syslog-mapping-cef.aspx) into Azure Sentinel. Refer to [Trend Micro Apex Central](https://docs.trendmicro.com/en-us/enterprise/trend-micro-apex-central-2019-online-help/preface_001.aspx) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **TMApexOneEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-TMApexOneEvent-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "TrendMicroApexOne",
+            "baseQuery": "TMApexOneEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All logs",
+            "query": "\nTMApexOneEvent\n| sort by TimeGenerated"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (TrendMicroApexOne)",
+            "lastDataReceivedQuery": "TMApexOneEvent\n | summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "TMApexOneEvent\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
```

Comment:
Can you please add "isPreview": true here?

Reply:
updated